### ISSUE TITLE
Building v6.0.1 on Windows fails

### ISSUE THREAD

**Issue by dsine-de** [barrysteyn/node-scrypt#121]
Hi,

building on Windows 10 fails with the new version of scrypt now on my box while v 5.4.1 worked.
My installed Visual Studio version is "Microsoft Visual Studio Express 2013 for Windows Desktop with Update 4", npm is v3.7.5 and Node.js is v5.7.0.

This is the log output (some german messages, but I can translate if you need more information):

```
PS E:\test> npm i scrypt

> scrypt@6.0.1 preinstall E:\test\node_modules\.staging\scrypt-f46119f4
> node node-scrypt-preinstall.js


> scrypt@6.0.1 install E:\test\node_modules\scrypt
> node-gyp rebuild


E:\test\node_modules\scrypt>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
Warning: Missing input files:
E:\test\node_modules\scrypt\build\..\scrypt\win\include\config.h
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  Copying scrypt/win/include/config.h to scrypt/scrypt-1.2.0/config.h
  Das System kann die angegebene Datei nicht finden.
  win_delay_load_hook.c
  Code wird generiert.
  Codegenerierung ist abgeschlossen.
  copied_files.vcxproj -> E:\test\node_modules\scrypt\build\Release\\copied_files.node
  memlimit.c
  keyderivation.c
  pickparams.c
  hash.c
  win_delay_load_hook.c
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(21): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(22): error C2085: 'libcperciva_be16dec': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(22): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(29): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(30): error C2085: 'libcperciva_be16enc': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(30): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(38): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(39): error C2085: 'libcperciva_be32dec': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(39): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(47): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(48): error C2085: 'libcperciva_be32enc': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(48): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(58): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(59): error C2085: 'libcperciva_be64dec': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(59): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(69): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(70): error C2085: 'libcperciva_be64enc': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(70): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(84): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(85): error C2085: 'libcperciva_le16dec': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(85): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(92): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_m odules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(93): error C2085: 'libcperciva_le16enc': Nicht in der formalen Parameterliste enthalten (..\src\scrypt wrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(93): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\n ode_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(101): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_ modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(102): error C2085: 'libcperciva_le32dec': Nicht in der formalen Parameterliste enthalten (..\src\scryp twrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(102): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\ node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(110): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_ modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(111): error C2085: 'libcperciva_le32enc': Nicht in der formalen Parameterliste enthalten (..\src\scryp twrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(111): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\ node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(121): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_ modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(122): error C2085: 'libcperciva_le64dec': Nicht in der formalen Parameterliste enthalten (..\src\scryp twrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(122): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\ node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(132): error C2054: Nach 'inline' muss '(' folgen (..\src\scryptwrapper\keyderivation.c) [E:\test\node_ modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(133): error C2085: 'libcperciva_le64enc': Nicht in der formalen Parameterliste enthalten (..\src\scryp twrapper\keyderivation.c) [E:\test\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
E:\test\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(133): error C2143: Syntaxfehler: Es fehlt ';' vor '{' (..\src\scryptwrapper\keyderivation.c) [E:\test\ node_modules\scrypt\build\scrypt_wrapper.vcxproj]
..\src\scryptwrapper\keyderivation.c(56): warning C4013: 'libcperciva_be32enc' undefiniert; Annahme: extern mit Rückgabetyp int [E:\test\node_modules\scrypt\build\scrypt_wrapper.v cxproj]
..\src\scryptwrapper\keyderivation.c(91): warning C4013: 'libcperciva_be32dec' undefiniert; Annahme: extern mit Rückgabetyp int [E:\test\node_modules\scrypt\build\scrypt_wrapper.v cxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:100:13)
gyp ERR! stack     at ChildProcess.emit (events.js:185:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 10.0.10586
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd E:\test\node_modules\scrypt
gyp ERR! node -v v5.7.0
gyp ERR! node-gyp -v v3.3.0
gyp ERR! not ok
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "i" "scrypt"
npm ERR! node v5.7.0
npm ERR! npm  v3.7.5
npm ERR! code ELIFECYCLE

npm ERR! scrypt@6.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the scrypt@6.0.1 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the scrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs scrypt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls scrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     E:\test\npm-debug.log
```

**Comment by barrysteyn** [comment]
Upgrade your compiler.

**Comment by dsine-de** [comment]
Could you give me a hint in the right direction on how to that on Windows 10? If I install Visual Studio 2015 (instead of 2013), it also uses ```C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe``` (or isn't that the compiler?).

**Comment by barrysteyn** [comment]
Hi @dsine-de 

If you can translate the error messages into English, I can give you more help.

**Comment by axic** [comment]
@barrysteyn we are experiencing the same issue, here's a gist with error messages in English: https://gist.github.com/anonymous/94cd1528fd8f95ba830ae7516ad40b92

```
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(21): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
  win_delay_load_hook.c
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(22): error C2085: 'libcperciva_be16dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(22): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(29): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(30): error C2085: 'libcperciva_be16enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(30): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(38): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(39): error C2085: 'libcperciva_be32dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(39): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(47): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(48): error C2085: 'libcperciva_be32enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(48): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(58): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(59): error C2085: 'libcperciva_be64dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(59): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(69): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(70): error C2085: 'libcperciva_be64enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(70): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(84): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(85): error C2085: 'libcperciva_le16dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(85): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(92): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(93): error C2085: 'libcperciva_le16enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(93): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(101): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(102): error C2085: 'libcperciva_le32dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(102): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(110): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(111): error C2085: 'libcperciva_le32enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(111): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(121): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(122): error C2085: 'libcperciva_le64dec' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(122): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(132): error C2054: expected '(' to follow 'inline' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(133): error C2085: 'libcperciva_le64enc' : not in formal parameter list (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
G:\workspace\testrpc\node_modules\scrypt\scrypt\scrypt-1.2.0\libcperciva\util\sysendian.h(133): error C2143: syntax error : missing ';' before '{' (..\src\scryptwrapper\keyderivation.c) [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
..\src\scryptwrapper\keyderivation.c(56): warning C4013: 'libcperciva_be32enc' undefined; assuming extern returning int [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
..\src\scryptwrapper\keyderivation.c(91): warning C4013: 'libcperciva_be32dec' undefined; assuming extern returning int [G:\workspace\testrpc\node_modules\scrypt\build\scrypt_wrapper.vcxproj]
```

**Comment by tcoulter** [comment]
Note: Just tested on Windows 8.1 64bit. Changing all `inline`’s to `__inline` in sysendian.h, cpusupport.h, and insecure_memzero.h fixes the build on Windows.

**Comment by axic** [comment]
@tcoulter good news, this was fixed in 6.0.2 (another issue detailing it #125)
